html/help.html

```diff
@@ -0,0 +1,30 @@
+<html>
+		<head>
```

Comment:
can we indent html with either 2 spaces, or only 1 tab? 2 tabs indents crazy far.